superbench/benchmarks/micro_benchmarks/sharding_matmul.py

```diff
@@ -255,7 +256,7 @@ def _benchmark(self):
                 return False
 
             metric = '{}'.format(mode)
-            if not self._process_numeric_result(metric, elapse_times):
+            if not self._process_numeric_result(metric, elapse_times, reduce_type=ReduceType.MAX):
```

Comment:
Why do we need to change Sharding Matmul?

Reply:
It depends on how many metrics do we want to get, for example:
For sharding matmul, I think we should get one metric no matter how many GPUs are used.
For kernel launch, we should get one metric per GPU.

Reply:
Make sense!